### PR TITLE
feat(android): Document audio capture bug

### PR DESCRIPTION
**Pull request by makohoek**
We received an e2e error for an audio capture bug.

Document the steps that are needed to solve it by providing a patch and links to other patches that need to be applied.

This is for @vishalmti to be applied on the older (SDK 10.0) documentation

### CODE REVIEW

File: source/devices/AM62PX/android/Release_Specific_Release_Notes.rst
Comment (StaticRocket) [inline source/devices/AM62PX/android/Release_Specific_Release_Notes.rst:125]: This patch is given in the e2e ticket already mentioned. Not sure it's worth adding to this repo as well. If it is accepted or in a public tree already I wouldn't mind you linking it here.

It's more of a personal opinion, but I don't like carrying temporary patches here. I'll let other maintainers comment though.

Comment (makohoek) [reply source/devices/AM62PX/android/Release_Specific_Release_Notes.rst:125]: The patch in question is only merged in https://gitlab.baylibre.com/baylibre/ti/android/aosp/external/tinyhal/-/commit/110fbd20bfd39fec96fcfecc87be3c497a2cb5c3

@vishalmti asked me to attach the patch since (afaik) it was not hosted on git.ti.com.

I can link to https://gitlab.baylibre.com/baylibre/ti/android/aosp/external/tinyhal/-/commit/110fbd20bfd39fec96fcfecc87be3c497a2cb5c3 if that is preferred, please let me know!

Comment (makohoek) [reply source/devices/AM62PX/android/Release_Specific_Release_Notes.rst:125]: I talked to @vishalmti on the internal chat and he agreed to add a link to our BayLibre gitlab commit. I will update this.

Comment (makohoek) [reply source/devices/AM62PX/android/Release_Specific_Release_Notes.rst]: Fixed in v2.

**Comment by makohoek** [comment]
Change in v3:
- Rebased on master

Change in v2:
- Remove .patch attachment

**Review by StaticRocket — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Comment by cshilwant** [comment]
@vishalmti Please review and Ack

**Review by jsuhaas22 — APPROVED** [review]

**Review by vishalmti — APPROVED** [review]

**Pull request merged**
